save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/contests/ContestParticipantsMenu.kt

```diff
@@ -72,7 +72,7 @@ private fun contestParticipantsMenu() = FC<ContestParticipantsMenuProps> { props
                 }
                 scoreCard {
                     name = "${contestResult.organizationName}/${contestResult.projectName}"
-                    contestScore = contestResult.score.toDouble()
+                    contestScore = contestResult.score ?: 0.0
```

Comment:
Well, according to @akuleshov7  this is what leetcode does. Actually it should be something like N/A, but it will be done in next PR. This PR is more about backend